documentation-support/src/main/java/tech/picnic/errorprone/documentation/Json.java

```diff
@@ -1,45 +1,25 @@
 package tech.picnic.errorprone.documentation;
 
-import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
-import com.fasterxml.jackson.annotation.PropertyAccessor;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.datatype.guava.GuavaModule;
-import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
-import com.google.errorprone.annotations.FormatMethod;
-import java.io.IOException;
-import java.io.UncheckedIOException;
 import java.nio.file.Path;
+import tools.jackson.databind.ObjectMapper;
+import tools.jackson.databind.json.JsonMapper;
+import tools.jackson.datatype.guava.GuavaModule;
 
 /**
  * Utility class that offers mutually consistent JSON serialization and deserialization operations,
  * without further specifying the exact schema used.
  */
 final class Json {
   private static final ObjectMapper OBJECT_MAPPER =
-      new ObjectMapper()
-          .setVisibility(PropertyAccessor.FIELD, Visibility.ANY)
```

Comment:
Now that we use records, this setting isn't necessary anymore. Also validated against the outstanding changes on the `website` branch.